packages/cli/src/metadataGeneration/typeResolver.ts

```diff
@@ -1033,7 +1060,7 @@ export class TypeResolver {
   }
 
   private getNodeDescription(node: UsableDeclaration | ts.PropertyDeclaration | ts.ParameterDeclaration | ts.EnumDeclaration) {
-    const symbol = this.current.typeChecker.getSymbolAtLocation(node.name as ts.Node);
+    const symbol = this.getSymbolAtLocation(node.name as ts.Node);
```

Comment:
Potentially wrong, as the private method getsymbolatlocation looks up the original Definition, but the node description is probably but the one we want to have from the current one

Reply:
Do you have an example where we can trigger a difference?

Reply:
I actually was playing around with it already and tried to find some difference, I couldn't find any issues. Therefore I think my comment can be silently closed ;-)

Reply:
I was wrongly assuming that this could be a problem:
![image](https://user-images.githubusercontent.com/5757263/135302252-6b426f86-17ba-43a0-ba8e-0edaa70b66b5.png)
but as we are looking for "CommonLink" and not the type of commonlink for the description itself, all is good.